src/index.js

```diff
@@ -1,9 +1,11 @@
 import {
-  Todo, todoForm, todoList,
+  Todo, todoForm, todoList, clearCompleteTodo,
 } from './modules/todo.js';
 import Views from './modules/views.js';
 import Persist from './modules/persist.js';
 // eslint-disable-next-line import/no-unresolved
+import Interactivity from './modules/interactivity.js';
```

Comment:
- [ ] Please fix your` " import Interactivity from './modules/interactivity.js';` as currently it is throwing this error, as there are no interactivity file in your modules directory, and when i try to remove this statement the error is gone but the status update is not working, Please fix this :point_down: 
![image](https://user-images.githubusercontent.com/97521557/209217706-d22d63f8-eb7d-4033-a079-3873e51705cd.png)